modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-verify_{context}"]
+== Verify cert-manager installation
+
+To verify cert-manager is installed, follow these steps:
+
+. Check `istiod` certificate:
++
+[source, terminal]
+----
+kubectl exec "$(kubectl get pods -n bookinfo -l app=productpage -o jsonpath='{.items[0].metadata.name}')" -c istio-proxy -n bookinfo -- \
+    openssl s_client -CAfile /var/run/secrets/istio/root-cert.pem -showcerts -connect istiod-basic.istio-system:15012 </dev/null
+----
+
+. Check app certificate:
++
+[source, terminal]
+----
+kubectl exec "$(kubectl get pods -n bookinfo -l app=productpage -o jsonpath='{.items[0].metadata.name}')" -c istio-proxy -n bookinfo -- \
+    openssl s_client -CAfile /var/run/secrets/istio/root-cert.pem -showcerts -connect details.bookinfo:9080 </dev/null
+----
```

Comment:
These verification steps are too low level. They are relevant for Istio developers, but not necessarily for users. Instead, I would recommend to just test traffic between applications in mesh with enabled strict mTLS.

Reply:
Can you remove this step?